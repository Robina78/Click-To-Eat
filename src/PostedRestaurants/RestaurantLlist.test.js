import React from "react";
import { render } from "@testing-library/react";
import RestaurantList from "./RestaurantList";


it("matches snapshot", function () {
  const { asFragment } = render(<RestaurantList />);
  expect(asFragment()).toMatchSnapshot();
});