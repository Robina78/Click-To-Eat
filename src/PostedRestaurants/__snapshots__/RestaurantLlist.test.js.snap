// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="RestaurantList mt-5 container"
  >
    <div
      class="col-lg-8 col-md-6"
      ms="12"
    >
      <div>
        <form
          class="form-search"
        >
          <input
            class="form-control input-search"
            name="searchTerm"
            placeholder="Enter restaurant name"
            value=""
          />
          <button
            class="btn btn-primary mb-4 ms-5 res-search"
            type="submit"
          >
            Search
          </button>
        </form>
      </div>
      <p
        class="lead"
      >
        Sorry, no results were found!
      </p>
    </div>
  </div>
</DocumentFragment>
`;
